framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -558,6 +558,9 @@ void MediaPlayerImpl::notifyObserver(player_observer_command_t cmd, ...)
 		case PLAYER_OBSERVER_COMMAND_PAUSED:
 			pow.enQueue(&MediaPlayerObserverInterface::onPlaybackPaused, mPlayerObserver, mPlayer);
 			break;
+		case PLAYER_OBSERVER_COMMAND_STOPPED:
```

Comment:
Please check below as a next step.
STOP means that player still look at the current song but all re-initialized. (If it is file, move to file position to at the start of file)
We using datasource, please find a way how we can reset current resources.
To sum up, stop means
1. Keep current track
2. Clean media buffer
3. Move to first position if it played file
4. Reset current frame number(not implemented yet.)

Reply:
@Taejun-Kwon Yes. I will modify it in the next commit.